samcli/lib/bootstrap/bootstrap.py

```diff
@@ -120,6 +120,27 @@ def _get_stack_template():
             - Key: ManagedStackSource
               Value: AwsSamCli
 
+      SamCliSourceBucketBucketPolicy:
+        Type: AWS::S3::BucketPolicy
+        Properties:
+          Bucket: !Ref SamCliSourceBucket
+          PolicyDocument:
+            Statement:
+              -
+                Action:
+                  - "s3:GetObject"
+                Effect: "Allow"
+                Resource:
+                  Fn::Join:
+                    - ""
+                    -
+                      - "arn:aws:s3:::"
+                      -
+                        !Ref SamCliSourceBucket
```

Comment:
this looks bit weird, should this be `- !Ref SamCliSourceBucket`?

Reply:
I think it could also be `!Sub "arn:aws:s3:::${SamCliSourceBucket}"`